backend/test/libs/util.spec.js

```diff
@@ -7,6 +7,27 @@ describe('util', () => {
   beforeEach(() => {
     config.network = 'dev';
   });
+
+  describe('isColor', () => {
+    it('should return true if it is valid', () => {
+      const reissuable =
+        'C1FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF';
+      assert.strictEqual(util.isColorId(reissuable), true);
+      const nonReissuable =
+        'C2FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF';
+      assert.strictEqual(util.isColorId(nonReissuable), true);
+      const nft =
+        'C3FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF';
+      assert.strictEqual(util.isColorId(nft), true);
+    });
+
+    it('should return false if it is invalid', () => {
+      const invalidColorId =
+        '00FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF';
+      assert.strictEqual(util.isColorId(invalidColorId), false);
+    });
+  });
+
```

Comment:
It would be better to have a test with over or under data length, data other than hex values.

Reply:
Add test case (a024c55)